device/transport/mqtt/package.json

```diff
@@ -46,20 +45,35 @@
   },
   "homepage": "https://github.com/Azure/azure-iot-sdk-node#readme",
   "nyc": {
-    "extension": [
-      ".ts",
-      ".tsx"
-    ],
     "exclude": [
+    	"coverage/**",
+      "packages/*/test{,s}/**",
       "**/*.d.ts",
-      "coverage/",
-      "index.js",
-      "test/"
+      "test{,s}/**",
+      "test{,-*}.{js,cjs,mjs,ts}",
+      "**/*{.,-}test.{js,cjs,mjs,ts}",
+      "**/__tests__/**",
+
+      "**/{ava,nyc}.config.{js,cjs,mjs}",
+      "**/jest.config.{js,cjs,mjs,ts}",
+      "**/{karma,rollup,webpack}.config.js",
+      "**/{babel.config,.eslintrc,.mocharc}.{js,cjs}",
+
+      "common.js",
+      "src/authentication_provider.ts",
+      "src/results.ts"
```

Comment:
Are we over specifying a bit her.  I don't think we have any babel or karma.  common.js?